exchanges/margin/margin_test.go

```diff
@@ -35,128 +29,90 @@ func TestUnmarshalJSON(t *testing.T) {
 	var alien martian
 	jason := []byte(`{"margin":"isolated"}`)
 	err := json.Unmarshal(jason, &alien)
-	if err != nil {
-		t.Error(err)
-	}
-	if alien.M != Isolated {
-		t.Errorf("received '%v' expected 'isolated'", alien.M)
-	}
+	assert.NoError(t, err)
+	assert.Equalf(t, Isolated, alien.M, "received '%v' expected '%v'", alien.M, Isolated)
```

Comment:
These assert.Equalf's are redundant since assert.Equal will just do it for you and makes it easier for you to write tests. So this and the below ones will become:

`assert.Equal(t, Isolated, alien.M)`

Reply:
<img width="613" alt="Screenshot 2025-01-22 at 4 53 13 in the afternoon" src="https://github.com/user-attachments/assets/968e7771-8445-4471-bd1b-a940f5b834a8" />

As shown in this picture, _assert.Equal()_ doesn't give the string representation when erroring. That's why I needed the _asset.Equalf()_ formatting.

Reply:
Thank you, Sir. I have done it according to your suggestion -- so -- we will take the blame together 😁